internal/commands/create.go

```diff
@@ -46,6 +41,37 @@ func newRegoFile(filePath string, contents string) (regoFile, error) {
 	return regoFile, nil
 }
 
+func newPolicyFile(filePath string, contents string) (regoFile, error) {
+	module, err := ast.ParseModule(filePath, contents)
+	if err != nil {
+		return regoFile{}, fmt.Errorf("parse module: %w", err)
+	}
+
+	if moduleHasViolationRules(module) == false {
+		return regoFile{}, nil
+	}
+
+	return newRegoFile(filePath, module, contents)
+}
+
+func newLibraryFile(filePath string, contents string) (regoFile, error) {
+	module, err := ast.ParseModule(filePath, contents)
+	if err != nil {
+		return regoFile{}, fmt.Errorf("parse module: %w", err)
+	}
+
+	return newRegoFile(filePath, module, contents)
+}
+
+func moduleHasViolationRules(module *ast.Module) bool {
```

Comment:
nit: really.. small.. nit. I would change this to singular because we're only checking for the existence of a single violation rule.